src/main/java/com/mindee/MindeeClient.java

```diff
@@ -53,6 +55,54 @@ public DocumentToParse loadDocument(
     return new DocumentToParse(fileInBase64Code, filename);
   }
 
+  public <T extends Inference> PredictResponse<T> jobStatusForId(Class<T> type, String jobId) {
+    return this.mindeeApi.checkJobStatus(type, jobId);
+  }
+
+  public <T extends Inference> String enqueue(Class<T> type, DocumentToParse documentToParse)
+      throws IOException {
+    return this.enqueue(type, documentToParse.getFile(), documentToParse.getFilename(),
+        Boolean.FALSE, null);
+  }
+
+  public <T extends Inference> String enqueue(
```

Comment:
The `enqueue` method should return the full response object.

Accessing the job's ID would be something along the lines of `enqueueResponse.getJob().getId()`.